walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -7,6 +7,8 @@
   </head>
   <body>
     <div id="content">
+      <p> #Jonathan Aner's Portfolio
```

Comment:
nit :remove the extra space before #. Do you intend to have the `#` before Jonathan?